pymedphys/_mosaiq/helpers.py

```diff
@@ -435,19 +445,20 @@ def get_all_treatment_data(cursor, mrn):
             "target_units",
             "technique",
             "modality",
-            "energy",
-            "fraction_dose",
-            "total_dose",
+            "energy [MV]",
+            "fraction_dose [cGy]",
+            "total_dose [cGy]",
             "fractions",
```

Comment:
How each of these column labels are linked into their respective "SELECT" items above is quite disjointed, making it hard to read, and I imagine quite easy to accidentally miss-align a column label to its data.

I could imagine something like the following being slightly more complicated in a programming sense, but nevertheless far easier to read:

```python
import collections

label_to_sql_table_and_column = collections.OrderedDict([
	("patient_id", "Ident.IDA"),	
    ("last_name", "Patient.First_Name"),
	("first_name", "Patient.Last_Name"),
	("date_of_birth", "Patient.Birth_DtTm"),
	# etc...
])

columns = label_to_sql_table_and_column.keys()
select_string = ", ".join(label_to_sql_table_and_column.values())

sql_string = # merge in select_string here...
```

Reply:
I definitely agree. I'll work on this ASAP.

Reply:
And thanks for the example btw!

Reply:
Having some difficulties with this. I've gotten to the point where sql_string looks identical to the sql command I was calling in execute_sql previously, but now it's not liking that. I will keep looking into this, but just sharing in case you've encountered this problem before.

![image](https://user-images.githubusercontent.com/40006991/89567430-4697eb80-d7e7-11ea-81d0-5c179e835dec.png)

Reply:
Is this better now?